erpnext/stock/doctype/product_recall_notice/product_recall_notice.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import json
+import frappe
+from frappe.model.document import Document
+
+class ProductRecallNotice(Document):
+	pass
+
+@frappe.whitelist()
+def create_stock_entry(product_recall_notice):
+	product_recall_notice = json.loads(product_recall_notice)
+	se_items = []
+	stock_entry_doc = frappe.new_doc("Stock Entry")
+
+	if product_recall_notice.get("recall_from") == "Warehouse":
+		stock_entry_doc.stock_entry_type = "Material Transfer"
+	elif product_recall_notice.get("recall_from") == "Customer":
```

Comment:
why is this an elseif? Can the recall_from field have values other than "Warehouse" or customer?